README.md

```diff
@@ -43,13 +43,22 @@ Install the following tools:
 
 * [vagrant](https://www.vagrantup.com/) (version 2.2.6)
 * [virtualbox](https://www.virtualbox.org/) (version 6.0)
+* [git bash](https://git-scm.com/downloads) (on windows only)
+
+**NOTE** - 
+_For the following instructions to work correctly on **Windows 10** you must install the latest version of **Git** and use **Git Bash** instead of **Windows CLI** to issue the commands._
```

Comment:
I'd much prefer to see Windows Subsystem for Linux being used instead of a third party git+bash packaging.

Reply:
Me too, but WSL 2.x and VirtualBox don't play nice together.